src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -139,7 +139,6 @@ def generate_ssns(
     size: int,
     additional_key: Any,
     randomness: RandomnessStream,
-    allow_duplicates: bool = False,
```

Comment:
The key difference here in the `generate_ssns` method is that I've removed the "allow_duplicates" arg b/c there's no use for it anymore (the only time this gets called is when a simulant is making an SSN up on the spot and duplicates are allowed in that case.